### ISSUE TITLE
Is it possible to record the 8-channel audio into pcm data (.raw file)?

### ISSUE THREAD

**Issue by chad1023** [respeaker/mic_array#30]
Is it possible to record the 8-channel audio into pcm data (.raw file)?

Thank you!

**Comment by xiongyihui** [comment]
If you use the respeaker usb 6+1 mic array, it can use the raw audio firmware to record 8 channel audio. See https://github.com/respeaker/mic_array_dfu

To record .raw audio, we can use `arecord -v -f S16_LE -c 8 -r 16000 -t raw -Dplughw:1 audio.raw`  (change `plughw:1` based on your respeaker sound card number)

**Comment by chad1023** [comment]
<img width="772" alt="2018-06-05 11 36 58" src="https://user-images.githubusercontent.com/13184977/40953964-de46d4b8-68b4-11e8-9c77-6d6c22bae9e9.png">

If we want to use the microphone array for recording, what should we set for plughw:1 ?

BTW, how could we controll the recording length? 
Is there the parameter for setting it?
Or what should we do for ending the recording? The program running and seem not to be stopped until I interrupted it with "Ctrl+C".
